packages/rum-core/src/domain/rumEventsCollection/action/getActionNameFromElement.ts

```diff
@@ -175,6 +175,18 @@ function getTextualContent(element: Element | HTMLElement) {
         }
       }
     }
+	
+	const elementsTextToScrub: NodeListOf<HTMLElement> = element.querySelectorAll(`[${DEFAULT_PROGRAMMATIC_ATTRIBUTE}]`)
+
+    // replace the innertext of the elements with programmatic attribute value
+    for (let i = 0; i < elementsTextToScrub.length; i += 1) {
+      const innerText = elementsTextToScrub[i].innerText
+      // eslint-disable-next-line @typescript-eslint/prefer-for-of
```

Comment:
Could you use `elementsTextToScrub.forEach`?

Reply:
Addressed. Replaced for with foreach